src/BlazorWebView/src/Maui/iOS/IOSWebViewManager.cs

```diff
@@ -74,10 +75,20 @@ public override void DidStartProvisionalNavigation(WKWebView webView, WKNavigati
 
 			public override void DecidePolicy(WKWebView webView, WKNavigationAction navigationAction, Action<WKNavigationActionPolicy> decisionHandler)
 			{
-				if (navigationAction.TargetFrame!.MainFrame)
+				// TargetFrame is null for navigation to a new window (`_blank`)
+				if (navigationAction.TargetFrame is null)
+				{
+					// Open in a new browser window
+					UIApplication.SharedApplication.OpenUrl(navigationAction.Request.Url);
+					decisionHandler(WKNavigationActionPolicy.Cancel);
```

Comment:
Could you elaborate on why this uses `WKNavigationActionPolicy.Cancel` vs `WKNavigationActionPolicy.Allow` in the default code path?

Reply:
We're cancelling the existing navigation event handled by the current Blazor iOS Web View Manager as we've already dispatched the event to be handled by the iOS browser. There's nothing further we want to do within the webview. This also follows a similar pattern to [what's discussed on SO](https://stackoverflow.com/a/36231713/17995448).